### PR TITLE
LPS-170571 | Add Autom. test ClayAlert#CanPreventTextFromDisplayingUnderIcon

### PR DESCRIPTION
**Pull request by deborasoliveira**
**ClayAlert#CanPreventTextFromDisplayingUnderIcon**
**Test Plan**

- Given Clay Sample Portlet
- When the alert contains class="alert-indicator"
- Then the text on the next line will not flow under the icon

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-170571

### CODE REVIEW

**Comment by deborasoliveira** [comment]
ci:test:frontend:nocompile

**Comment by deborasoliveira** [comment]
cc: @john-co, @mateusralv

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlert.testcase [hunk 1]
```diff
@@ -74,6 +74,30 @@ definition {
 		}
 	}
 
+	@description = "Verify the text on the next line will not flow under the icon on the alert"
+	@priority = "4"
+	test CanPreventTextFromDisplayingUnderIcon {
+		task ("When the alert contains class='alert-indicator'") {
+			VerifyElementPresent(
+				key_alertType = "success",
+				locator1 = "ClayAlert#ALERT_INDICATOR");
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlert.testcase:83]: Hey @deborasoliveira could you change `VerifyElementPresent` for `AssertElementPresent`?

**Review by mateusralv — CHANGES_REQUESTED** [review]
Only minor modifications, congratulation Debora 😃

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlert.testcase (hunk 1 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlert.testcase:83]: It's better to have VerifyElementPresent in When steps. We don't want to have any assertions in Given or When step if possible.

Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlert.testcase:83]: Reason is, we are more focused on the Then steps failing and we may not care if something fails in the Given or When step b/c other tests should be covering those steps if we built our SACRED model correctly.

**Review by john-co — APPROVED** [review]
Looks good to me! :sunglasses:

Nice job @deborasoliveira!

**Comment by john-co** [comment]
ci:forward

**Pull request closed without merging**